src/xkit.cs

```diff
@@ -1861,7 +1861,7 @@ interface NSMutableParagraphStyle {
 		NSTextTableBlock [] TextBlocks { get; set; }
 #endif
 
-		[iOS (16, 0), TV (16, 0), NoWatch, MacCatalyst (16, 0)]
+		[NoWatch, MacCatalyst (13, 1)]
 		[NoMacCatalyst]
```

Comment:
this isn't really a part of this pr's changes, but why do we have conflicting attrs here? based on docs it is available starting from mac cat 13.1+, so seems a bit weird having `[NoMacCatalyst]` here as well?

Reply:
That looks like an old c&p mistake. I'll fix it too.